node/src/chain/acceptor.rs

```diff
@@ -579,6 +593,41 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
     pub(crate) async fn get_outbound_chan(&self) -> AsyncQueue<Message> {
         self.task.read().await.outbound.clone()
     }
+
+    async fn adjust_round_base_timeout(&self) -> TimeoutSet {
```

Comment:
Use the plural: `adjust_round_base_timeouts`
Also, if you rename the variable to `base_timeouts`, rename this to `adjust_base_timeouts`